components/Landing/HeroMoment/LandingHeroMomentComponent.vue

```diff
@@ -94,23 +137,68 @@ const qiskitPronunciation = Math.random() < 0.5 ? "[kiss-kit]" : "[quiss-kit]";
 
     @include carbon.breakpoint-down(md) {
       background-image: none;
+      padding-top: carbon.$spacing-05;
     }
 
     &__image {
-      height: 100%;
+      height: auto;
+      width: 60%;
       position: absolute;
-      right: 0;
+      right: 9%;
       top: 0;
       z-index: -1;
 
-      @include carbon.breakpoint-down(md) {
+      @include carbon.breakpoint-down(lg) {
         display: none;
       }
     }
+
+    &__laptop {
+      width: 793px;
```

Comment:
We usually don't work with such specific sizes but rather proportional, responsive sizes. Is there a need to do it like this in this specific case? This also goes for the other very specific size definitions in these styles.